tests/test_cli.py

```
@@ -346,129 +314,63 @@ def count_models(self, query: ModelCatalogQuery) -> int:
-            errors_sampled=[],
-            artifact_path=artifact_path,
-        )
-
-    monkeypatch.setattr(cli_module, "_repo", fake_repo_builder)
-    monkeypatch.setattr(cli_module, "run_repository_benchmark", fake_run_benchmark)
+
+    class _FakeClient:
+        def __init__(self, *, registry: object, repository: object | None) -> None:
+            _ = registry, repository
+
+        def query(self, request, **kwargs):  # noqa: ANN001, ARG002
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `query`

(ANN202)